models/ticketingDao.js

```diff
@@ -0,0 +1,50 @@
+const appDataSource = require('./dataSource');
+
+const getMovies = async () => {
+  try {
+    const rawQuery = `
+    SELECT DISTINCT
+      m.id as id,
+      m.name as movieTitle,
+      m.thumbnail_image_url as movieThumbnail,
+      DATE_FORMAT(m.opening_date, '%Y-%m-%d') as releaseDate,
+      fr.rating as rating
+    FROM movies m
+    JOIN film_ratings fr ON m.film_rating_id = fr.id
+    ORDER BY m.name ASC;
+    `;
+    const result = await appDataSource.query(rawQuery);
+    return result;
+  } catch (err) {
+    throw err;
+  }
+};
+
```

Comment:
```suggestion
const getMovies = async () => {
  const rawQuery = `
    SELECT DISTINCT
      m.id as id,
      m.name as movieTitle,
      m.thumbnail_image_url as movieThumbnail,
      DATE_FORMAT(m.opening_date, '%Y-%m-%d') as releaseDate,
      fr.rating as rating
    FROM movies m
    JOIN film_ratings fr ON m.film_rating_id = fr.id
    ORDER BY m.name ASC;
  `;
  return appDataSource.query(rawQuery);
};

```
1. catchAsync로 controller 함수를 깜싸주면, controller는 service를, service는 dao를 가져다 사용하기 때문에 마찬가지로 try catch가 필요하지 않습니다.
2. 변수 선언 없이 곧바로 return 해도 되는 흐름의 코드입니다.

다른 코드들도 모두 적용 및 수정해주세요~